src/gesture-handler/GestureDetectorProvider.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { GHContext } from 'react-native-screens';
+import { GHContext } from '../native-stack/contexts/GHContext';
```

Comment:
I would be afraid of doing it in the separate package, same for `native-stack`. But if you checked it outside the repo and it works, then I'm fine with that 🎉

Reply:
What are the possible dangers here?

Reply:
I'm not sure but we followed this convention for some reason I think 😅

Reply:
It might have been auto-imported like that during development, that's what eslint suggests.
I built a clean app with screens installed from this branch just to make sure it works, it does 😄